sources/data-node/grpc-rest-bindings.yml

```diff
@@ -144,3 +144,15 @@ http:
       get: '/oracle-specs/{id}/oracle-data'
     - selector: datanode.api.v1.TradingDataService.OracleSpecs
       get: '/oracle-specs'
+
+    # Nodes
+    - selector: api.v1.TradingDataService.GetNodeData
+      get: '/nodes/data'
+    - selector: api.v1.TradingDataService.GetNodes
+      get: '/nodes'
+    - selector: api.v1.TradingDataService.GetNodeByID
+      get: '/nodes/{id}'
```

Comment:
I suspect `/nodes/data` and `/nodes/{id}` are going to collide, because "data" could be interpreted as an ID.

See also `/markets/{id}` vs `/markets-data` (which would be nicer as `/markets/data`) in https://github.com/vegaprotocol/protos/blob/develop/sources/vega/grpc-rest-bindings.yml.